### DISCUSSION TITLE
Missing Windows installer for 4.8.2alpha1

### DISCUSSION THREAD

**Discussion by jcasale** [texstudio-org/texstudio#3746] [Q&A] [answered]
Hi,
The [4.8.2alpha1](https://github.com/texstudio-org/texstudio/releases/tag/4.8.2alpha1) release does not contain a Windows installer, is that an isolated occurrence with just this release?

**Comment by muzimuzhi** [comment, marked as answer]
Should be fixed in [4.8.3alpha3](https://github.com/texstudio-org/texstudio/releases/tag/4.8.2alpha3) pre-release.
